handlers/lib/sshConfig.js

```diff
@@ -6,7 +6,8 @@ module.exports.getSSHConfig = () => {
     host: getEnv("SFTP_HOST"),
     port: getEnv("SFTP_PORT"),
     username: getEnv("SFTP_USER"),
-    privateKey: getEnv("SFTP_PRIVATE_KEY"),
+    password: getEnv("SFTP_PASSWORD", ["SFTP_PRIVATE_KEY"]),
+    privateKey: getEnv("SFTP_PRIVATE_KEY", ["SFTP_PASSWORD"]),
```

Comment:
I believe it's not an issue if *both* password and SSH key is set, in that case what's used depends on the server.